templates/hardware.html

```diff
@@ -165,12 +165,12 @@ <h2>Hardware details</h2>
                   {% if "/" not in device.identifier %}
                     {# / breaks the component URLs #}
                     {% if device.subsystem %}
-                      <a href="/catalog/component/{{ device.subsystem }}/{{ device.identifier }}">{{ device.name }}</a>
+                      <a href="/catalog/component/{{ device.subsystem }}/{{ device.identifier }}">{{ device.make }} {{ device.name }}</a>
```

Comment:
Here you have code in the app that is already prepending make. It is where device_info is built